cnf-certification-test/networking/suite.go

```diff
@@ -67,8 +73,59 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 	ginkgo.It(testID, ginkgo.Label(testID), func() {
 		testMultusNetworkConnectivity(&env, defaultNumPings, netcommons.IPv6)
 	})
+	// Default interface ICMP IPv6 test case
+	testID = identifiers.XformToGinkgoItIdentifier(identifiers.TestUndeclaredContainerPortsUsage)
+	ginkgo.It(testID, ginkgo.Label(testID), func() {
+		testListenAndDeclared(&env)
+	})
 })
 
+//nolint:funlen
+func testListenAndDeclared(env *provider.TestEnvironment) {
+	var k declaredandlistening.Key
+	failedContainers := []string{}
+
+	for _, cut := range env.Containers {
+		declaredPorts := make(map[declaredandlistening.Key]*provider.Container)
+		listeningPorts := make(map[declaredandlistening.Key]*provider.Container)
+		ports := cut.Data.Ports
+		fmt.Println(cut)
+		logrus.Debugf("%s declaredPorts: %v", cut.StringShort(), ports)
+		if ports == nil {
+			tnf.ClaimFilePrintf("%s doesn't not have any declared port", cut.StringShort())
```

Comment:
I agree to remove the "if ports== nil" but I think that we need to stay this statement **" logrus.Debugf("%s declaredPorts: %v", cut.StringShort(), ports)"** because this getting information which ports is declared or this not interesting? @edcdavid